graviti/dataframe/indexing.py

```diff
@@ -80,6 +110,55 @@ def __getitem__(self, key: Iterable[int]) -> "DataFrame":
     def __getitem__(self, key: Union[int, Iterable[int]]) -> Any:
         return self.obj._getitem_by_location(key)
 
+    @overload
+    def __setitem__(self, key: int, value: Iterable[Any]) -> None:
+        pass
+
+    @overload
+    def __setitem__(self, key: Union[Iterable[int], slice], value: Iterable[Iterable[Any]]) -> None:
+        pass
+
+    def __setitem__(
+        self,
+        key: Union[int, Iterable[int], slice],
+        value: Union[Iterable[Any], Iterable[Iterable[Any]]],
+    ) -> None:
+        pass
+
+    def extend(self, objs: Iterable[Union[Dict["str", Any], "DataFrame"]]) -> None:
```

Comment:
why we need to add extend rows to loc and iloc, and they look exactly the same. Could we add extend_rows to dataframe?